
# Abstract {.page_break_before}
The Human Cell Atlas (HCA) is an international initiative that aims to comprehensively characterize every cell type in the human body. 
HCA members employ advanced techniques, such as single-cell RNA sequencing, mass cytometry, and multiplexed in situ hybridizations, to generate cell-level data from virtually all human tissues. The vast amount of knowledge derived from this wealth of data requires effective management to maximize the societal benefits of HCA.

While semantic technologies have emerged as key players in the data management ecosystem, there are still gaps that need to be addressed. 
Wikidata, a sister project of Wikipedia focused on structured data, is emerging as a central hub in the semantic web, connecting experts and the general public. 
By integrating the Human Cell Atlas with Wikidata, HCA can seamlessly incorporate its products into the broader knowledge ecosystem, thereby extending its reach and impact.

This PhD project aims to investigate Wikidata as a platform for representing cell types. 
To optimize the curation process, we have developed Wikidata Bib, a framework for managing scientific literature and organizing notes effectively.
Using Wikidata Bib, we have reviewed the literature and curated thousands of cell classes in a structured format within Wikidata. 
Additionally, we have mapped numerous terms from the Cell Ontology to the Wikidata infrastructure and are actively working on integrating Wikidata into Cell Ontology workflows. 
This integration will enable the Human Cell Atlas curators to smoothly incorporate the curated information into their daily activities.

Finally, we explored the linked open data infrastructure of Wikidata to construct metascientific knowledge graphs for the bioinformatics community, showcasing the applications of Wikidata beyond the representation of scientific concepts and delving into the intricate workings of science itself.


# Project summary  {.page_break_before}


The emergence of single-cell technologies has amplified the interest of the scientific world in life's fundamental components â€“ the cells [@wikidata:Q99418649].
Leading the charge in the realm of cell knowledge is the Human Cell Atlas (HCA) project, initiated in 2017 with the ambitious objective of characterizing every cell type in the human body [@wikidata:Q46368626].
This consortium, encompassing contributors from across the globe, includes significant contributions from Brazil, where Prof. Helder Nakaya (the supervisor of this PhD project) is instrumental in the national effort to support HCA, particularly focusing on the context of infectious and inflammatory diseases.

The HCA is a game-changer for the biomedical sciences, crafting tools and setting standards for basic research, and facilitating a comprehensive characterization of diseases, ultimately enhancing diagnostics and therapeutic strategies. 
For this large reaching effort to be a complete success, its data products need embody the FAIR principles: being findable, accessible, interoperable, and reusable [@wikidata:Q27942822].
With an increased emphasis on data stewardship and data management, the scientific community is rapidly adapting to these core demands, improving the standards used to organize of scientific outputs.


The Human Cell Atlas has a dedicated team for organizing data: the Data Coordination Platform (DCP; [https://data.humancellatlas.org/about](https://data.humancellatlas.org/about])) [@wikidata:Q104450645].
The DCP is responsible for tracing the plan for computational interoperability, from the data generators to the consumers.[@wikidata:Q104450645].
In addition to its core team, the HCA is poised to grow by community interaction, and states in its opening paper that "As with the Human Genome Project, a robust plan will best emerge from wide-ranging scientific discussions and careful planning".[@wikidata:Q46368626]  
Thus, this project inserts itself among the wide-ranging scientific discussions to improve data - and knowledge - interoperability. 


With the vast volume of data generated by the HCA, efficient knowledge management strategies are crucial.
One approach includes extensive use of ontologies to annotate datasets and consolidate information. Ontologies, in this context, are precise computational representation of reality, attempting to encapsulate concepts and their relationships accurately [@wikidata:Q105870680]. 
These formal concepts are used by data annotators to improve the interoperability of data.
For instance, cellxgene, a prominent platform for single-cell RNA-seq data analysis from the Chan Zuckerberg Initiative, utilizes OBO Foundry ontologies to standardize annotations of cell types, anatomical structures, and other biological entities (refer to the curation schema at <https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md>).



Probably the most renowned biomedical ontology, the Gene Ontology, describes various classes of biological processes, interlinked by "is_a" and "part_of" relations [@wikidata:Q104130127; @wikidata:Q23781406].
The Gene Ontology is part of the broader effort of the Open Biomedical and Biological Ontologies (OBO) Foundry to formalize concepts across biology [@wikidata:Q19671692].
Established in 2007, the OBO Foundry is a central repository of biomedical ontologies that provides guidelines for designing and constructing high-quality ontologies.
The initial OBO Foundry unified several independent ontologies like the Cell Ontology (CL), the Disease Ontology (DO), and the Protein Ontology (PRO) under a common framework, marking significant progress towards interoperability. 
Among the many ontologies relevant to the HCA, the Cell Ontology (the ontology of cell types) unarguably plays a central role [@wikidata:Q109755180].


<!-- Add paragraph on the usage of the Cell Ontology -->

The Cell Ontology was first described in 2005 [@wikidata:Q21184168] and was oriented both at creating a species-neutral classification of cells and
for researchers to "learn a considerable amount about that cell type and its relationships to other biological objects".[@wikidata:Q21184168]
The collaborative project gradually evolved and changed its design and scope to fit new needs. 
By 2011, the demand for computable definitions resulted in an increase in the number and quality of immune cell types represented in CL [@wikidata:Q33786317].
It further incorporated species-specific cell types to better manage marker-based definitions typically presented at the species level [@wikidata:Q33786317].
Over the years, the development of the Cell Ontology included both technical enhancements and the addition of new cell types.
As of the last official CL publication in 2016, it featured approximately 2,200 classes [@wikidata:Q36067763].
Today, the Cell Ontology continues to expand as a resource for the Human Cell Atlas, providing identifiers for cell types [@wikidata:Q109755180], and now encompasses over 2,500 cell classes.

While ontologies are powerful tools, they demand a high level of technical expertise to get started.
Recently, the dawn of collaborative knowledge graphs has given rise to a new methodology for formal knowledge representation.
Wikidata, the collaborative knowledge graph of the Wikimedia Foundation, encourages users to contribute classes and statements, aligning with Wikipedia's ethos of communal wisdom and its "epistemic virtues" such as power, speed, and availability [@wikidata:Q101955295].

The power of Wikidata stems from its large community of contributors.
With a community boasting over 46,000 active monthly editors and attracting 420 million views per month, it covers an expansive range of concepts far beyond the reach of any individual user [@wikidata:Q118130164].
Wikidata is also speedy because it requires no software installation or permissions to update: any user can simply make changes via a web interface.
This speed makes it more accessible for newcomers to join and contribute, a stark contrast to the OBO Foundry ontologies that necessitate extensive semantic training and knowledge of Git/GitHub for contributions.
Lastly, Wikidata's information is readily available through a user interface, a SPARQL query service, and as comprehensive database dumps, ensuring full-scale reusability.
Its CC0 public domain license waiver guarantees this reusability in all settings [@wikidata:Q118130164].
The success of the Wikidata model led Google to migrate its own knowledge base, Freebase, entirely into Wikidata [@wikidata:Q24074986].
Wikidata has now been integrated into workflows by major organizations like Amazon, OpenAI, Twitter, Apple, The Met and Smithsonian museums, and more [@wikidata:Q118130164].

So far, several advancements have been made towards biological data integration and analysis in Wikidata, resulting in positive outcomes and demonstrating its potential for bioinformatics-related analyses, such as drug repurposing and ID conversion [@wikidata:Q21503284; @wikidata:Q87830400].
With over 50,000 human gene items indexed and hundreds of biomedical-related properties, Wikidata has been proposed as a unified base for collecting and disseminating biomedical knowledge [@wikidata:Q68471881].


This project's goal is to bridge the progress of collaborative knowledge modeling on Wikidata with the scientific endeavor of the Human Cell Atlas.
We are reviewing the single-cell literature, refining and formalizing concepts for cell type delineation, and exploring their application in the Wikidata database.
Simultaneously, we are investigating how the Wikidata knowledge system can be integrated with the OBO Foundry to improve the coverage and quality of multiple ontologies.
The objectives of the project have evolved over time and currently can be better summarized as:

* Analyze and supplement the community-built data model to describe human cell classes on Wikidata.
* Develop a curation workflow for cell types on Wikidata and provide a comprehensive catalog of cell types.
* Study the intersections of Wikidata and bioinformatics in general, and the Human Cell Atlas in specific, and how the platform can intersect with the needs of scientific stakeholders.
* Develop tools and standard operating procedures to connect Wikidata and the OBO Foundry, as a stepping stone to improve the Human Cell Atlas curation efforts.