
# Abstract {.page_break_before}
The Human Cell Atlas (HCA) is an international effort to characterize every cell type of the human body. 
HCA members are producing cell-level data from virtually all human tissue using techniques such as single-cell RNA sequencing, mass cytometry, and multiplexed in situ hybridizations. 
The wealth knowledge stemming from the wealth of data needs management to maximize the societal benefit of HCA.
While semantic technologies have emerged as key players in the data interoperability ecosystem, there are still gaps to bridge. 
Wikidata, a sister project of Wikipedia for structured data, is surfacing as a hub in the semantic web for multiple types of information and bridging experts and the people. 
The connection of the Human Cell Atlas to Wikidata can quickly insert its products into the larger knowledge ecosystem, extending HCA's reach. 
This PhD project aims at studying Wikidata as a platform for representing cell types.  
We review the literature on cell types, refining and formalizing concepts for cell type delimitation.
At the same time, integrate biomedical resources (e.g. PanglaoDB amd the Cell Ontology) into the Wikidata infrastructure and we enrich Wikidata with  manually curated cell types. 
We are also improving the interplay of Wikidata, the Cell Ontology and software used for single-cell RNA-seq data, inserting Wikidata _de facto_ as a tool for the Human Cell Atlas community. 
To optimize the curation, we developed Wikidata Bib, a framework for literature management and organized note-taking system for dealing with the academic literature, as well the wdcuration package. 
Finally, we leverage the linked open data infrastructure of Wikidata to build metascientific knowledge graphs of the bioinformatics community, showcasing its applications beyond the representation of scientific concepts and into the inner workings of science.

# Project summary  {.page_break_before}

The advent of single-cell technologies has deepened the scientific community's interest in the building blocks of life, the cells [@wikidata:Q99418649]. 
The Human Cell Atlas (HCA) project, has been a major player in the cell knowledge ecosystem, running since 2017 towards the task to characterize every cell type in the human body [@wikidata:Q46368626]. The HCA consortium recruited people from all over the world to tackle different parts of the project. In Brazil, Prof. Helder Nakaya (supervisor of this PhD project) is playing a key role the national effort to contribute to HCA, with a focus on the roles of different cell types in the pathological processes of infectious and inflammatory diseases.

HCA is revolutionizing the biomedical sciences, by creating tools and standards for basic research, as well as allowing better characterization of disease, and thus, ultimately, improving diagnostics and therapy. 
Its products (data, information, knowledge and wisdom) need to be FAIR: findable, accessible, interoperable and reusable.
Data stewardship and data management are growing as core demands of the scientific community, ranging from data management plans [@wikidata:Q56524391] to specialized personnel [@wikidata:Q56524391].

The Human Cell Atlas has a dedicated team for organizing data: the Data Coordination Platform (DCP) [@url:https://data.humancellatlas.org/about] [@wikidata:Q104450645].
The DCP is responsible for tracing the plan for computational interoperability, from the data generators to the consumers.[@wikidata:Q104450645].In addition to its core team, the HCA is poised to grow by community interaction, and states in its opening paper that "As with the Human Genome Project, a robust plan will best emerge from wide-ranging scientific discussions and careful planning".[@wikidata:Q46368626]  
Thus, this project inserts itself among the wide-ranging scientific discussions to improve data - and knowledge - interoperability. 

The large amount of information generated by HCA creates the need for efficient knowledge management approaches. One of them is the wide appliation of ontologies to annotate datasets and summarize information, with a key role of the Cell Ontology (the ontology of cell types) in the ecosystem. [@wikidata:Q109755180]. For example, cellxgene  is a leading platform for single-cell RNA-seq data analysis from the Chan Zuckerberg Initiative, and it employs OBO Foundry ontologies 
to standardize annotations of cell types, anatomical structures and other biological entities."  [@url:"https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md"]. An ontology, as used here, is a formal computational representation of reality, which tries to represent each concept (and their relations) as precisely as possible.  [@wikidata:Q105870680]  

The Gene Ontology is probably the most well known biomedical ontology; it describes (among other things) different classes of biological process, related to each_other by "is_a" and "part_of relations. [@wikidata:Q104130127] [@wikidata:Q23781406]. The Gene Ontology is part of a much larger effort to formalize concepts across biology: the Open Biomedical and Biological Ontologies (OBO) Foundry. [@wikidata:Q19671692]
Created in 2007, the OBO Foundry is a hub of biomedical ontologies that sets guidelines for the design and construction of high-quality ontologies. 
The initial OBO Foundry united several independent ontologies, such as the Cell Ontology (CL), the Disease Ontology (DO) and the Protein Ontology (PRO) under a common framework, a great progress towards interoperability. 

<!-- Add paragraph on the usage of the Cell Ontology -->

The Cell Ontology was first described in 2005 [@wikidata:Q21184168] and was oriented both at creating a species-neutral classification of cells and
for researchers to "learn a considerable amount about that cell type and its relationships to other biological objects".[@wikidata:Q21184168]
The collaborative project gradually evolved and changed its design and scope to fit new needs. 
By 2011, a need for computable definitions led to an advance in the number and quality of immune cell types represented in CL. [@wikidata:Q33786317] 
It also included the addition of species-specific cell types to handle better marker-based definitions, usually presented at the species level. [@wikidata:Q33786317] 
Further developments over the years included both technical improvements and the addition of new cell types. 
By the time of the last official CL publication, in 2016, it contained approximately 2,200 classes.   [@wikidata:Q36067763]  
Currently, the Cell Ontology is growing as a resource for the Human Cell Atlas and in providing identifiers for cell types [@wikidata:Q109755180] and presents over 2500 classes of cells.


Ontologies are powerful, but require a high degree of technical expertise to get started. 
Recently, a new approach for formal knowledge representation arose with the dawn of collaborative knowledge graphs.
Wikidata, the collaborative knowledge graph of the Wikimedia foundation, allows users to contribute with classes and statements, in the same spirit of Wikipedia and share its "epistemic virtues, like power, speed and availability. [@wikidata:Q101955295]
It is powerful because of its large community of contributors. With a community of more than 46,000 active monthly editors and 420 million monthly views growing, it is able to cover a much wider number of concepts than any user individually. [@wikidata:Q118130164]
It is fast, because one does not need to install any software or ask for permissions to update it: any user can simply do it via a web interface. 
That speed makes it easier for newcomers to join and contribute, in contrast to OBO Foundry ontologies, which require extensive training on semantics and knowledge of Git/GitHub for contributions. 
Finally, the information on Wikidata is available via an user interface, via a SPARQL query service and as large, full-size database dumps, providing full extent reusability. Its CC0 public domain license waiver guarantees this reusability in any settings.[@wikidata:Q118130164]
The Wikidata model has been so sucessfull that Google decided to migrate its own knowledge base, Freebase, fully into Wikidata.[@wikidata:Q24074986] Wikidata has now been integrate into workflows by organizations like Amazon, OpenAI, Twitter, Apple, The Met and Smithsonian museums and more. [@wikidata:Q118130164]

Several advances towards biological data integration and biological data analysis in Wikidata have been made so far, yielding positive results [@doi:10.1101/031971] [@wikidata:Q87830400] and showcasing its potential for bioinformatics-related analyses, such as drug repurposing and ID conversion [@doi:10.7554/eLife.52614]. 
Wikidata has been proposed as a unified base to gather and distribute biomedical knowledge, with more than 50 000 human gene items indexed and hundreds of biomedical-related properties [@doi:10.1016/j.jbi.2019.103292].


This project aims to connect the advancements of collaborative knowledge modeling on Wikidata to the Human Cell Atlas scientific endeavor. We are reviewing the single-cell literature, refining and formalizing concepts for cell type delimitation, and exploring their application in the Wikidata database. At the same time, we are exploring how the Wikidata knowledge system can be integrated to the OBO Foundry to improve the coverage and quality of multiple ontologies. The specific goals outlined in the approved project were:

* Build a data model to capture the main properties to describe a human cell. Provide working definitions of cell types and states and their characteristics.
* Extract from and add to Wikidata pieces of information regarding Human Cell Atlas publications to build gold standards. Use this information to develop machine learning tools to extract knowledge from publications.
* Create tools to make data from the underlying knowledge graph accessible by employing methods from network theory.

These goals evolved in time and currently can be better summarized as:

* Analyze and complement the community-built data model to capture the main properties to describe a human cell on Wikidata 
* Study the concept of “cell type” and “cell state” in the context of data curation 
Develop a curation workflow for cell types on Wikidata and provide a comprehensive catalog of cell types 
* Study the intersections of Wikidata and bioinformatics in general, and the Human Cell Atlas in specific, and how the platform can intersect with the needs of scientific stakeholders.
* Develop tools and standard operating procedures to connect Wikidata and the OBO Foundry, as a stepping stone to improve the Human Cell Atlas curation efforts

