

## Biocuration of Cell Classes using Wikidata Bib {.page_break_before}
### Introduction

Annotation skills are a crucial part of formal academic training in humanities [@wikidata:Q118385266].
A seminal work outlining methods for academic reading in humanities is Umberto Eco's book "How to Write a Thesis" [@isbn:9780262527132].
While it provides significant insights into the annotation of literature for academic theses, its pre-digital era scope and focus on humanities somewhat limit its application to the biomedical sciences.

The need for an organized reading system for biocuration studies arises from a difference in methodology.
In humanities, written text is the primary research material [@wikidata:Q118385266].
However, in biomedical sciences, including a significant portion of bioinformatics, the natural world observed through experimentation is the object of study.
As the number of scientific articles continues to grow exponentially, the scientific literature and accompanying public datasets have become a substantial resource for shaping scientific projects.
Therefore, the development of a methodology for academic reading tailored to the digital environment is an emerging necessity.

In response, we developed Wikidata Bib, a Wikidata-based framework for large-scale curation of scientific articles.
The framework leverages Wikidata's metascientific coverage, which includes structured metadata for tens of millions of scientific articles and tools for adding new articles as needed.

### Methods

#### Wikidata Bib as a Reading System 

Wikidata Bib's reading framework is built on a git repository integrated with GitHub, Python scripts, and SPARQL queries.
The code is packaged into a Python module to facilitate usage, using the Click library to provide a Command Line Interface for end-users (<https://github.com/pallets/click>).
After installing the package using the pip utility, users can use Wikidata Bib from the terminal like any other command-line utility.
The basic commands available are as follows:

* `bib read`: Accepts a Wikidata QID for an article, opens the article's web page in a browser, and opens a personalized notes document. It also updates the static files for the analytics dashboard, stored in the docs/ directory.
* `bib pop`: "Pops" an article from a reading stash (manually and semi-automatically curated) at toread.yml and runs `bib read` for it.
* `bib log`: Updates the git repository and adds, commits, and pushes notes and the dashboard to GitHub.

Wikidata Bib is coupled with a discipline of daily reading, inspired by Robert Cecil Martin's description of Test Driven Development in the book "Clean Code" [@isbn:9780132350884]. 
Over the past two years since the initial usable version of Wikidata Bib, I've maintained a routine of reading one article about cell types (for example, single-cell RNA-seq studies) and one article about biocuration (for instance, articles describing ontologies) each day.
I've taken notes on every article using the notetaking station depicted in Figure @fig:notetaking, and these notes are also accessible in the GitHub repository. 
This consistent reading practice allows for steady coverage of relevant literature. While this discipline has been effective for this research project, it is not a requirement for using the Wikidata Bib system.

In addition to providing a method for organized reading, the Wikidata Bib framework also seeks to improve metadata about articles on Wikidata whenever possible. 
In Figure {@fig:notetaking}B, the links included in the dashboard are shown.
A link to a Scholia [@wikidata:Q41799194] profile allows the identification of related articles from a series of pre-made SPARQL queries probing bibliography data on Wikidata.
While Scholia provides an overview of a given article, it does not allow direct curation of the metadata. 
To address this, two links are provided: one to the Wikidata interface and one to the Author Disambiguator (<https://www.wikidata.org/wiki/Wikidata:Tools/Author_Disambiguator>). 
Accessing the Wikidata page for the entity allows the addition of new triples, for instance, curating authors and topics of the article, which are then used by Scholia and by Wikidata Bib's dashboard. 
The AuthorDisambiguator is a tool that simplifies the process of disambiguating author names to unique identifiers on Wikidata, thereby enriching the public knowledge graph of publications and authors.

Finally, a link to the article's DOI or full-text URL is provided and serves as a fallback when the automatic download fails. Importantly, while the metadata curation has a technical benefit to Wikidata and the dashboard, it also plays a theoretical role. By curating metadata on authors, the user of Wikidata Bib can better understand the people behind the research they read, thereby expanding their metascientific perspective on their domain of interest. Moreover, by integrating metascientific curation into the reading process, Wikidata Bib encourages the user to familiarize themselves with the individuals producing the science they read, thus contributing to a more comprehensive perspective of the research process.

![Wikidata Bib's platform for note taking.](https://raw.githubusercontent.com/lubianat/phd_figures/master/wikidata_bib/note_taking_station_annotated_with_links.png){#fig:notetaking  width="80%"}

The source code for Wikidata Bib is available at <https://github.com/lubianat/wikidata_bib>. 


#### Wikidata Bib for curation of cells to Wikidata

The Wikidata Bib system was originally devised to allow an overview of the single-cell RNA-seq and biocuration fields. 
However, during the process, we repurposed it for the biocuration of new cell classes in Wikidata. 
By fast-tracking reading, Wikidata Bib enables an efficient parsing of the literature and, thus, the identification of previously uncatalogued cell types.
Articles read with Wikidata Bib were screened for mentions of cell types absent from Wikidata. 
We considered a "cell type" as any class of cells described by a domain expert with evidence of the reality of its instances. 
When a mention of such a class appears in an article, I first verify Wikidata for the existence of a related class. 
If that the class is absent from the platform, I enter a new class name (usually the name chosen by the domain expert in the reference article), alongside a superclass, and a QID in a Google Spreadsheet, as shown in Figure @fig:biocuration_of_cells.

![Wikidata Bib was coupled with a biocuration framework for cell types.](https://raw.githubusercontent.com/lubianat/phd_figures/master/wikidata_bib/biocuration_of_cells.png){#fig:biocuration_of_cells width="85%"}

The information from the spreadsheet is pulled by a python script and processed locally with a series of dictionaries that match common terms to Wikidata IDs. 
In the example shown in Figure @fig:biocuration_of_cells, the string "endothelial cell" in the spreadsheet was matched against a manually curated dictionary to the Wikidata entry "[Q11394395](https://www.wikidata.org/wiki/Q11394395)", which represents that concept on Wikidata. 
After reconciling the data, the script uses the Wikidata Integrator python package (<https://github.com/SuLab/WikidataIntegrator>) to insert the new entries on the Wikidata database. 
The code for integrating a Google Spreadsheet to Wikidata is available at <https://github.com/lubianat/wikidata_cell_curation>. 


<!-- UPDATE THE STATISTICS 
Jupyter Notebook: https://colab.research.google.com/drive/1GvQXOs51_U8icdGMtKXMeLOXKM8pXWet
-->

### Results

![Statistics on cell type information on Wikidata as of 20/04/2023. A) Number of cell type items divided by taxon (P703, found in taxon, relation). The top 5 is shown. B) Number of cell type items created by each Wikidata user. The top 10 are shown. C-D) Total number of modifications of cell type items by each user, either total number (C) or the number of different edited items (D). The top 10 are shown.](https://raw.githubusercontent.com/lubianat/phd_figures/master/wikidata_cell_curation/statistics_20230420.png){#fig:wikidata_statistics_on_cell_classes width="85%"}

After the intense curation activity performed in this PhD project, Wikidata now contains 6102 subclasses of "cell ([Q7868](https://www.wikidata.org/wiki/Q7868))" as of 20 of April of 2023 (Figure @fig:all_cells_network and Figure @fig:wikidata_statistics_on_cell_classes). 
From those, 550 cell classes are specific for humans, and 318 are specific for mice.  
Currently Wikidata has more cell classes than the Cell Ontology, which lists around 2700 classes. 
It is worth noticing that classes on the Cell Ontology are added after careful consideration by ontologists and domain experts and should be considered of higher quality than the ones on Wikidata. 

From the 6102 cell classes on Wikidata, 5981 (98.0%) have been edited somehow by User:TiagoLubiana, and 4686 (76.8%) have been created by User:TiagoLubiana. 
Edits included adding multilanguage labels, connecting a dangling Wikipedia page to the cell subclass hierarchy, adding identifiers, images, markers, and other pieces of information. 
Approximatedly 430 hundred terms were added via manual curation based on PanglaoDB entries, while the remaining 4256 entries were created either via Wikidata's web interface or via the curation workflow described in this chapter. 

These statistics are demonstration of how the curation system used during this PhD project efficiently contributes to the status of cell type information on Wikidata.
Far from a flat curation, the cell classes on Wikidata were connected, when feasible, to each other and to other entities. 
Figure @fig:all_cells_network depics the network for just one relation, "subclass of" (P279), and already demonstrates the extensiveness of the graph.

![Overview of the subclass structure for cells in the Wikidata knowledge graph. The labels for the 10 nodes with highest degree are shown.](https://raw.githubusercontent.com/lubianat/phd_figures/master/wikidata_cell_curation/subclass_hierarchy_on_wikidata_20230512.png){#fig:all_cells_network}


The use of tens of Wikidata properties to describe those cells leads a large, multifaceted knowledge graph.
 Figure @fig:top_relations_for_cell_types.png shows a schema for the most used properties in the platform and exemplifies the kinds of knowledge that can be formalized in the system.
Each of those relations can be queried in Wikidata's SPARQL endpoint, enabling complex, multi-hop queries navigating the biological landscape of Wikidata.
A set of SPARQL queries showcasing different questions now askable in the Wikidata graph can be seen at <https://lubianat.github.io/cell_type_query_book/>.
Furthermore, these cell classes are linked to identifiers in a variety of external platforms, providing a cross mapping and acquiring the role of a knowledge hub (see Table @tbl:wikidata-ids-for-cells)


Given this live nature, the contributions of this project to the Wikidata platform in the context of the life sciences are already "live".
They are part of the web of knowledge, both directly and undirectly, as sources such as OpenAI, Apple's Siri and the Google Knowledge Graph are known to use Wikidata in its operations. [@wikidata:Q118130164]
Wikidata's stable funding structure and long-term planning, alongside the copy-friendly CC0 license, guarantee the digital preservation of the knowledge curated, and its findability, accessibility, reusability and interoperability (FAIRness) for the foreseeable future.



| property | propertyLabel                           | counts |
|----------|-----------------------------------------|--------|
| wd:P7963 | Cell Ontology ID                        | 2635   |
| wd:P1402 | Foundational Model of Anatomy ID         | 889    |
| wd:P672  | MeSH tree code                           | 542    |
| wd:P6366 | Microsoft Academic ID                    | 505    |
| wd:P646  | Freebase ID                              | 387    |
| wd:P10283| OpenAlex ID                              | 277    |
| wd:P486  | MeSH descriptor ID                       | 269    |
| wd:P1417 | Encyclop√¶dia Britannica Online ID        | 201    |
| wd:P3827 | JSTOR topic ID                           | 150    |
| wd:P1694 | Terminologia Histologica                 | 149    |

Table: Top 10 links to identifiers community curated on cell classes on Wikidata. The identifier with larger coverage  is the Cell Ontology ID, designed and curated in the course of this PhD project. {#tbl:wikidata-ids-for-cells}


![Top 10 relations used in Wikidata to represent cell classes as of May 12, 2023. The number of triples using each of the relations is shown under parenthesis.](https://raw.githubusercontent.com/lubianat/phd_figures/master/wikidata_cell_curation/top_10_properties_20230512.png){#fig:top_relations_for_cell_types.png}


However, even if Wikidata presents on itself a nice framework for community-encoding of knowledge into structured format, it would be naive to assume the Human Cell Atlas community would start using the platform in the short term.
Even though libraries and museums are increasingly using Wikidata as a source of identifiers [@wikidata:Q105429450], the life sciences community is still in the infancy of dealing with structure knownledge. 
Wikidata lacks the authoritativeness of a domain-specific governing body, such as the OBO Foundry, the Human Genome Nomenclature Consortium (HGNC), the International Catalog of Diseases (ICD), MeSH, and other label-and-identifier-providers for the life sciences.
Thus, while Wikidata plays a role in the knowledge representation ecosystem (with applications outside its own platform, e.g. as a mint for identifiers to use in nanopublications [@wikidata:Q111483757; @wikidata:Q118383816]), we believe the more immediate contribution of Wikidata to the Human Cell Atlas will come indirectly, by improving the curation workflow in ontologies and providing insights into the metascience of events and communities. 
Those relations will be discussed in the upcoming sessions.


<!-- UPDATE THE STATISTICS -->

