
## Biocuration of cell classes with Wikidata Bib  {.page_break_before}
### Introduction 

Annotation skills are part of formal academic training in the humanities  [@url:https://bibliotecadaeca.wordpress.com/2019/09/30/como-fazer-um-fichamento][@url:"https://www.youtube.com/playlist?list=PLAudUnJeNg4vWJhEJ_da26C-QW5qiS7uZ"]. 
An influential work in presenting methods for academic reading in the humanities is Umberto Eco's book "How to Write a Thesis" [@wikidata:Q3684178], which outlines not only _how_ to annotate the literature that basis an academic thesis, but also _why_ to do so. 
The book, written originally in 1977, is still influential today. 
Still, its theoretical scope (roughly the humanities) and its date preceding the digital era limits the extent to which it applies to the biomedical sciences. 

Notably, the need for an organized reading system for biocuration studies stems from a difference in methodology. 
In humanities, the main (if not sole) research material is the written text, the books and articles from which research stems—[@url:"https://www.youtube.com/playlist?list=PLAudUnJeNg4vWJhEJ_da26C-QW5qiS7uZ"].
In the biomedical sciences, including a large part of bioinformatics, the object of study is the natural world, observed via experimentation. 
Thus, naturally, scientific training focuses on experimentation and data analysis's theoretical and practical basis. 
With the boom of scientific articles, however, the scientific literature (and accompanying public datasets) already provide a strong material for sculpting scientific projects.
Thus, developing a methodology for academic reading tailored to the digital environment is a need. 

To address this gap, we developed  Wikidata Bib, a Wikidata-based framework for large scale curation of scientific articles. 
The framework is based on the metascientific coverage of Wikidata, as the platform also contains structured metadata for tens of millions of scientific articles, and tooling for adding new articles on demand.
### Methods
#### Wikidata Bib as a reading system 

The reading framework of Wikidata Bib is built upon a git repository integrated with GitHub, Python scripts, and SPARQL queries. 
The code is packaged into a python module to facilitate usage, using the Click library for providing a Command Line Interface for end-users (https://github.com/pallets/click).
After installing the package using the pip utility (https://pypi.org/), the user can use Wikidata Bib from the terminal as any other command-line utility. 
The basic commands available are described below:
 * `bib read` receives a Wikidata QID for an article and opens the article web page on a browser and opens a personalized notes document. It also updates the static files for the analytics dashboard, stored in the docs/ directory.
 * `bib pop`, which “pops” an article from a reading stash (manually and semi-automatically curated) at toread.yml and runs `bib read` for it 
 * `bib log` updates the git repository and adds, commits, and pushes notes and the dashboard to GitHub

The Wikidata bib framework is coupled with a discipline of daily reading.
The discipline is inspired by Robert Cecil Martin's description of Test Driven Development in the book "Clean Code", which includes not only a technical description but a _school of thought_ of how software development might be approached.  [@wikidata:Q109996684]
For the 2 years since the first usable version of Wikidata Bib, I've read one article about cell types (e.g. single-cell RNA-seq studies) and one article about biocuration (e.g. articles describing ontologies). I took notes on every article using the notetaking station displayed in Figure @fig: notetaking, and these notes are also made available in the GitHub repository.
The constancy of reading allows steady coverage of the relevant literature. 
While the discipline has worked for this research project, it is not required to use the Wikidata Bib system.

The Wikidata Bib framework also includes, whenever possible, an improvement of the metadata about the article on Wikidata. 
In @fig: notetaking B are shown the links included in the dashboard. 
A link to a Scholia [@wikidata:Q41799194] profile allows identification of related articles from a series of pre-made SPARQL queries probing bibliography data on Wikidata.
While Scholia provides an overview of a given article, it does not allow direct curation of the metadata.
For that, two links are provided, one to Wikidata and one to Author Disambiguator [@url:https://www.wikidata.org/wiki/Wikidata:Tools/Author_Disambiguator].
By accessing the Wikidata page for the entity, one can add new triples, for example, curating authors and topics of the article, which are then used by Scholia and by Wikidata Bib's dashboard. 
Author Disambiguator is a wrapper of an Wikimedia API that facilitates disambiguating author names to unique identifiers on Wikidata, thus feeding the public knowledge graph of publication and authors.  
Finally, a link to the article's DOI or full-text URL is provided and serves as a fallback when the automatic download fails. 
Of note, while the metadata curation has a technical benefit to Wikidata and the dashboard, it also plays a theoretical role. 
By curating metadata on authors, the user of Wikidata Bib can better understand the people they read, and expand their metascientific perspective on their domain of interest. 
Furthermore, by embedding the metascientific curation in the reading process WikidataBib directs the user to get acquainted with the people producing the science they read, thus contributing to a more complete perspective of the research process.

![Wikidata Bib's platform for note taking](https://raw.githubusercontent.com/lubianat/phd_figures/master/wikidata_bib/note_taking_station_annotated_with_links.png){#fig:notetaking}

The source code for Wikidata Bib is available at <https://github.com/lubianat/wikidata_bib>. 


#### Wikidata Bib for curation of cells to Wikidata

The Wikidata Bib system was originally devised to allow an overview of the single-cell RNA-seq and biocuration fields. 
However, during the process, we repurposed it for the biocuration of new cell classes in Wikidata. 
By fast-tracking reading, Wikidata Bib enables an efficient parsing of the literature and, thus, the identification of previously uncatalogued cell types.
Articles read with Wikidata Bib were screened for mentions of cell types absent from Wikidata. 
We considered a "cell type" as any class of cells described by a domain expert with evidence of the reality of its instances. When a mention of such a class appears in an article, I first verify Wikidata for the existence of a related class. 
Suppose that the class is absent from the platform. 
In that case, I enter a new class name (usually the name chosen by the domain expert in the reference article), alongside a superclass, and a QID in a Google Spreadsheet, as shown in @fig:biocuration_of_cells.

The information from the spreadsheet is pulled by a python script and processed locally with a series of dictionaries that match common terms to Wikidata IDs. 
In the example shown in Figure @fig:biocuration_of_cells, the string "endothelial cell" in hte spreadsheet was matched against a manually curated dictionary to the Wikidata entry [Q11394395](https://www.wikidata.org/wiki/Q11394395), the representation of that concept on Wikidata. 
After reconciling the data, the script uses the Wikidata Integrator python package [@url:https://github.com/SuLab/WikidataIntegrator] to insert the new entries on the Wikidata database. 
The code for integrating a Google Spreadsheet to Wikidata is available at <https://github.com/lubianat/wikidata_cell_curation>. 

![Wikidata Bib was coupled with a biocuration framework for cell types](https://raw.githubusercontent.com/lubianat/phd_figures/master/wikidata_bib/biocuration_of_cells.png){#fig:biocuration_of_cells width="85%"}

<!-- UPDATE THE STATISTICS 
Jupyter Notebook: https://colab.research.google.com/drive/1GvQXOs51_U8icdGMtKXMeLOXKM8pXWet
-->

### Results
![Statistics on cell type information on Wikidata as of 20/04/2023. A) Number of cell type items divided by taxon (P703, found in taxon, relation). The top 5 is shown. B) Number of cell type items created by each Wikidata user. The top 10 are shown. C-D) Total number of modifications of cell type items by each user, either total number (C) or the number of different edited items (D). The top 10 are shown.](https://raw.githubusercontent.com/lubianat/phd_figures/master/wikidata_cell_curation/statistics_20230420.png){#fig:wikidata_statistics_on_cell_classes width="85%"}

After the intense curation activity performed in this PhD project, Wikidata now contains 6102 subclasses of "cell ([Q7868](https://www.wikidata.org/wiki/Q7868))" as of 20 of April of 2023 (@fig:all_cells_network, @fig:wikidata_statistics_on_cell_classes). 
From those, 550 cell classes are specific for humans, and 318 are specific for mice.  
Currently Wikidata has more cell classes than the Cell Ontology, which lists around 2700 classes. 
It is worth noticing that classes on the Cell Ontology are added after careful consideration by ontologists and domain experts and should be considered of higher quality than the ones on Wikidata. 

From the 6102 cell classes on Wikidata, 5981 (98.0%) have been edited somehow by User:TiagoLubiana, and 4686 (76.8%) have been created by User:TiagoLubiana. 
Edits included adding multilanguage labels, connecting a dangling Wikipedia page to the cell subclass hierarchy, adding identifiers, images, markers, and other pieces of information. 
Approximatedly 430 hundred terms were added via manual curation based on PanglaoDB entries, while the remaining 4256 entries were created either via Wikidata's web interface or via the curation workflow described in this chapter. 

These statistics are demonstration of how the curation system used during this PhD project efficiently contributes to the status of cell type information on Wikidata.
Far from a flat curation, the cell classes on Wikidata were connected, when feasible, to each other and to other entities. 

![Overview of the subclass structure for cells in the Wikidata knowledge graph. The labels for the 10 most connect nodes are shown.](https://raw.githubusercontent.com/lubianat/phd_figures/master/wikidata_cell_curation/subclass_hierarchy_on_wikidata_20230512.png){#fig:all_cells_network}


The use of tens of Wikidata properties to describe those cells forms a large, multifaceted knowledge graph.
A sample of the expressiveness of the platform can be seem in figure @fig:top_relations_for_cell_types.png, which shows a schema for the most used properties in the platform.
Each of those relations can be queried in Wikidata's SPARQL endpoint, enabling complex, multi-hop queries navigating the biological landscape of Wikidata.
Furthermore, these cell classes are linked to identifiers in a variety of external platforms, providing a cross mapping and acquiring the role of a knowledge hub.


Thus, the contributions of this project to the Wikidata platform in the context of the life sciences is already "live" and actively contributing to the web of knowledge, both directly and undirectly, as sources such as OpenAI, Apple's Siri and the Google Knowledge Graph are known to use Wikidata in its operations. [@wikidata:Q118130164]
Wikidata's stable funding structure and long-term planning, alongside the copy-friendly CC0 license, guarantee the digital preservation of the knowledge curated, and its findability, accessibility, reusability and interoperability (FAIRness) for the foreseeable future.

![Top 10 relations used in Wikidata to represent cell classes as of May 12, 2023. The number of triples using each of the relations is shown under parenthesis.](https://raw.githubusercontent.com/lubianat/phd_figures/master/wikidata_cell_curation/top_10_properties_20230512.png){#fig:top_relations_for_cell_types.png}

| property | propertyLabel                           | counts |
|----------|-----------------------------------------|--------|
| wd:P7963 | Cell Ontology ID                        | 2635   |
| wd:P1402 | Foundational Model of Anatomy ID         | 889    |
| wd:P672  | MeSH tree code                           | 542    |
| wd:P6366 | Microsoft Academic ID                    | 505    |
| wd:P646  | Freebase ID                              | 387    |
| wd:P10283| OpenAlex ID                              | 277    |
| wd:P486  | MeSH descriptor ID                       | 269    |
| wd:P1417 | Encyclopædia Britannica Online ID        | 201    |
| wd:P3827 | JSTOR topic ID                           | 150    |
| wd:P1694 | Terminologia Histologica                 | 149    |

Table X: Top 10 links to identifiers community curated on cell classes on Wikidata. The identifier with larger coverage  is the Cell Ontology ID, designed and curated in the course of this PhD project.


However, even if Wikidata presents on itself a nice framework for community-encoding of knowledge into structured format, it would be naive to assume the Human Cell Atlas community would start using the platform in the short term.
Even though libraries and museums are increasingly using Wikidata as a source of identifiers

ADD REFERENCES

the life sciences community is still in the infancy of dealing with structure knownledge. 
Wikidata lacks the authoritativeness of a domain-specific governing body, such as the OBO Foundry, the Human Genome Nomenclature Consortium (HGNC), the International Catalog of Diseases (ICD), MeSH, and other label-and-identifier-providers for the life sciences.
Thus, while Wikidata plays a role in the knowledge representation ecosystem (with applications outside its own platform, e.g. as a mint for identifiers to use in nanopublications)

ADD REFERENCE

as we will discuss in the next sessions, we believe the more immediate contribution of Wikidata to the Human Cell Atlas will come indirectly, by improving the curation workflow in ontologies and providing insights into the metascience of events and communities. 


<!-- UPDATE THE STATISTICS -->

