## Mapping the metascience on bioinformatics onto Wikidata {.page_break_before}

### Introduction

During the development of this project we came to realize the potential of Wikidata to represent not only scientific concepts, but also metascientific concepts. 
Besides capturing information about the world, Wikidata has been developed as a platform to capture structured data on the sources for encoded statements . 
In other words, Wikidata became a citation repository, containing structured information on tens of millions of scientific articles.
This project was spearheaded by WikiCite, an organization dedicated to improve citations in the context of Wikimedia projects. 
The information encoded on Wikidata powers the provenance infrastructure of the platform, enabling queries to, for example, list the articles that were used as source of the cell types in the platform (@fig:cell_type_sources).

![Articles and databases most used as source for curation of cell types on Wikidata](ADD_SOURCE_LINK){#fig:cell_type_sources width="85%"}

The modelling of metadata about articles gradually expanded in scope, and Wikidata developed infrastructure to model other aspects of the publication landscape: theses, preprints, authors, institutions, research topics, academic software, conferences and more. 
One project, Scholia, rised as a platform showcasing Wikidata, SPARQL-based automatic profiles for different entities in academia. 

ADD REFERENCE

The insights provided by Scholia, such as co-author graphs and topic scores for institutions, triggered us to think about the biological knowledge graph in Wikidata in a broader sense, including not only the scientific concepts but also the players in the bioknowledge ecosystem.    

Given this potential of the platform, we invested time in expanding tooling and coverage of Wikidata for bioinformatics information, with a focus in the Brazilian bioinformatics community and in the context of the Human Cell atlas. 
In this chapter, we describe the PyOrcidator package, to integrate information from the ORCID platform to Wikidata on demand, a project to map the Brazilain bioinformatics landscape to Wikidata, a semantic catalog of USP Bioinformatics thesis and, finally an example of how to use the Wikidata infrastructure to get insights on particular academic events.


### Methods

Our process of building a metascientific knowledge graph and then displaying it to the community consisted of several steps, from data modelling to visualiazion. 
After identifying an analytical scope, we manually criated the core Wikidata entries for entities, adding aspects such as the location and parent institutions of programs, and start and end dates for academic events.  
Then, we used webscraping in Python to recover lists of related people, and manually reconciled the names to Wikidata, using both WDCuration and a set of other tools, such as the Wikipedia and Wikidata add-on for Google Sheets

ADD LINK

 and the Open Refine data curation platform.

ADD LINK

For each person, we run the PyOrcidator tool (described below) to update information on Wikidata, as well as other tools such as AuthorDisambiguator to disambiguate articles to their authors. 
Finally, to visualize the information, we wrote a series of SPARQL queries and/or used Scholia's standard SPARQL queries, to generate insights and visualizations on the graph.
Due to the ever-updating nature of the curation effort, we  made sure that all subprojects had  online, live dashboards either independently on GitHub Pages or on the Scholia platform.

#### PyOrcidator

One of the most important aspects of biomedical knowledge is the biomedical researcher. 
By having accurate metadata on people, we empower our community to better tailor its projects to the society.
For example, actionable metadata on affiliations, gender and (if public sources are available) ethnicity can help Equity, Diverstiy and Inclusion (EDI) efforts. 
In the context of the Human Cell Atlas, it can help us identifying biases and gaps in publications and events, and thus tweak the actions around the community to help the project to reach its goals. 
It can also help academic departments and programs to know more about its academic output, and use this data for improvement.

Currently, ORCID has risen as the core international standardized provider of academic curricula for individuals. 

ADD REFERENCE

Even though Currículo Lattes is the standard for academic curricula in Brazil, we focused on ORCID for its better international coverage, its open Application Program Interface, and its clear CC0 (public domain dedication) license, which enables reuse of the metadata. 
The ORCID API provides semi-structured information for every researcher, including DOIs for the listed publications. 
Most information provided, however, is in the form of natural language, meaning that users enter their affiliations and topics of research in free-format fields. 
That feature makes it harder, for example, to query ORCID for all members of a particular research institution, as names may be written in a myriad of different ways. 

The PyOrcidator tool combines parsing of the ORCID API with dictionary handling using WDCuration, enabling curators to map, on demand, the free labels of ORCID to the numeric identifiers of Wikidata.
It stores tree dictionaries, one for roles (e.g. "PhD Student", "Lead developer"), one for institutions (e.g. "University of São Paulo", "FioCruz Foundation") and one for topics (e.g. "Cell biology", "bioinformatics").
Users of the package clone the GitHub repository with the containing dictionaries, perform curation on demand and push the changes back to GitHub.
This decentralized curation enables gradual mapping of institution labels to Wikidata, effectively allowing reuse of curation effort.

For each ORCID, PyORCIDator generates a file in the Quickstatements 

ADD REFERENCE

format containing the triples that will be updated in the knowledge graph, including cross-identifiers present in ORCID (e.g. to Twitter and ResearchGate), the affiliations and education information, the topics studied and the publications for the author.
The PyORCIDator package is instrumental for the mapping of academic programs and conferences to Wikidata, enabling targeted curation of the metadata for the researchers of interest.

### The Landscape of Brazilian Bioinformatics

To exercise Wikidata as a framework for cataloguing scientific metadata, we set ourselves to the task of designing a map of the stakeholders in Brazilian bioinformatics. 

We created Wikidata entries for several bioinformatics programs in Brazil, and mapped the researchers affiliated to each of them on Wikidata. 
We also assumend that any researcher affiliated to a core bioinformatics program could also receive a triple stating "field of work --> bioinformatics".
For determining the people in scope of the project, we decided to filter for the following constraints: 

 1. People that have the triple field of work --> bioinformatics
 2. People that have some relation (employed by | affiliated to | educated at) an institution that has the triple "country --> Brazil". 

 We decided on that loose mapping to reach bioinformaticians in Brazil widely, even those that are not affiliated to a bioinformatics-only academic program. 
 Besides our individual curation of the academic programs, we organized a 2-day virtual event in november 2023 that gathered people from accross Brazil to share their perspectives and aid in the curation effort. 

The final product of the mapping is a live dashboard on the landscape of Brazilian bioinformatics that can be accessed at
https://lubianat.github.io/bioinfo_brasil/dashboard, highlighted in figure @fig:landscape_of_bioinformatics. 
The dashboard contains a series of SPARQL query results showcasing different facets of the Brazilian bioinformatics, such as the giant co-authorship component (@fig:landscape_of_bioinformatics A-B) with few separate communities, and the bias towards United States and European institutions for study and work. 


![Selected visualizations from the dashboard of the landscape of Brazilian bioinformatics. A) The co-authorship network of all brazilian bioinformatician (according to the previously outlined criteria). B) An inset of the graph, showing the high connectivity of the University of São Paulo community. Images are displayed whenever available under an open license in the Wikimedia Commons Database. For researchers without images, women are shown in green and man are shown in orange. Other genders or missing information would appear in white. C) The map of past an present affiliations of Brazilian bioinformaticians. All images displayed are screenshots of SPARQL query results available with live-updated data at https://lubianat.github.io/bioinfo_brasil/dashboard,](https://raw.githubusercontent.com/lubianat/phd_figures/master/metascience_on_wikidata/landscape_of_brazilian_bioinformatics.png){#fig:landscape_of_bioinformatics width="85%"} 

#### USP Bioinformatics Thesis

Besides the mapping of the bioinformatics researcher, we dedicated time to model and gather insights on the theses and dissertations produced by students at the University of São Paulo Graduate Program in Bioinformatics (also available at https://lubianat.github.io/bioinfo_brasil/dashboard)
Metadata for each publications was parsed from the Teses USP repository into a curation framework, where committee members and topics were curated from free text into Wikidata identifiers. 
Each publication, then, was created as a new item on Wikidata containing the reconciled metadata, making them available for querying. 

In @fig:bioinformatics_theses A, we see "bioinformatics" as the common main topic, but also concepts such as "transcriptome", "metagenomics" and "single-nucleotide polymorphisms", denoting the variety of subjects within the scope of the program.  In @fig:bioinformatics_theses B, we note that Prof. Helder Nakaya, the supervisor of this project, is the person that most attended defenses of the program. This data does not discern between own students and invitations, and thus this bias needs to be taken into account. 

@fig:bioinformatics_theses C presents a co-participation in bioinformatics defense committee network. 
We reasoned that, similar to co-authorship graphs, a committee co-participation network could bring insights into the soft structures of science. 
In this network, the presence of smaller, disconnected subcomponents, such as the one highlighted in @fig:bioinformatics_theses D, highlight different communities of practice.
In this case, the 3 professors of the program in the subgraph, Prof. Júlia Pavan, Prof. Anatoli Iambartsev  and Prof. Antonio Galves are all from the Institute of Mathematics and Statistics, perhaps signaling a difference in the type of research performed and evaluated.


Besides gathering factual evidence, the open metascientific maps on Wikidata have the power to show light at hidden structures of science. 
It provides a democratic way to navigate the landscape of academic affinities: which people publish together, which ones don't? Which professors participate in committee thesis together? 
Obvious applications are the rational improvement of structures and the aid to detect fraudulent schemes. But the non-obvious applications are important too: by encoding the metascientific structure openly, newcomers and young scientists have a way to understand the inner details of the academic carreer, and thus have a better shot at succeeding. 

![Topics and committee members of USP Bionformatics Theses and dissertations. A) The top 10 topics via Teses USP of the thesis in the program. B) The 5 people that most participated in defenses of the program C) A co-participation in committee graph for the USP Bioinformatics program. Researchers affiliated to the program are shown in blue. D) An inset of the graph, showing a community disconnected from the giant component. Graphs are not directional, arrows are an artifact of the graph visualization in the Wikidata Query Service.](https://raw.githubusercontent.com/lubianat/phd_figures/master/metascience_on_wikidata/bioinformatics_theses.png){#fig:bioinformatics_theses width="85%"}


### Event-based curation 

With the end of the COVID-19 pandemic health emergency, academic conferences started to happen again. 
Academic events are at the core of modern scientific practice, providing grounds for mix and match of people, and new projects that combine expertises to advance science. 
The structuring of event-level information using state-of-the-art Linked Open Data standards can help participants, organizers and funders to benefit the most of those events. 
Co-author graphs of participants and speakers, for example, can help us to identify opportunities for collaboration. 
Metainformation on location and gender, on the other hand, can auxiliate organizers to improve events in terms of diversity.


#### Natal Bioinformatics Forum 2023

For this report, we took the example of a knowledge graph of the speakers at the Natal Bioinformatics Forum, an academic gathering that happend in Natal, Rio Grande do Norte, in March 2023. 

As with the academic programs, the information for speakers was scraped, reconciled to Wikidata, and curated on the platform, and made available in a dashboard at https://lubianat.github.io/natal_bioinformatics2wikidata/. 
In @fig:natal_bioinformatics A, we can observe the co-authorship relations of the speakers in the event. 
Prof. Ana Tereza Vasconcelos stems as a hub in the graph, providing bridges accross subcomponents. 
The meaning of this relations was made clear during her talk at the event - Prof. Ana Tereza is one of the pioneers in bioinformatics in Brazil, and has been publishing on the field for decades, with wide ranging collaborations.

The mapping of the event to Wikidata also enable a quick overview of the kinds of software the speakers used in their publications. 
Besides the imaging software ImageJ, we could see a bias towards R packages for dealing with transcriptomics, such as "edgeR", "DESeq2", "limma" and "affy", also illustrating a bit the inner practices of the community. 
Finally, the map of affiliations of the speakers shows a clear bias towards United States and Europe. 
Strikingly, Latin American institutions were absent from the academic history of the speakers, and even inside Brazil, we see 2 localized clusters, one in the south-southwest and the other in a small part of the northwest.
The affiliations maps are powerful tools for identifying geographic biases and, hopefully, addressing them in the future. 


![An overview of the speakers at the Natal Bioinformatics Forum 2023. A) The co-authorship network of the speakers at the event, excluding speakers with no connections. B) A bubble plot showing the pieces of software most used by the spearkers in publications, according to Wikidata. C) The map of past and present affiliations of the speakers at the event.](https://raw.githubusercontent.com/lubianat/phd_figures/master/metascience_on_wikidata/natal_bioinformatics.png){#fig:natal_bioinformatics width="85%"} 


### Discussion 

#### Contextualization with the Human Cell Atlas

While the metascientific analysis described here are of relevance to Brazilian bioinformatics, we reason that the same tools can be used to understand, and address where needed, gaps in the Human Cell Atlas. 
We have started curating the metadata of Human Cell Atlas Publications

ADD LINK

to Wikidata, creating a new possibility to look at biases (geographic and otherwise) in the project. 
This is specially important in the context of HCA, as one of its core missions is to provide an atlas that is representative of the human diversity. 
We are also currently mapping the speakers and poster presenters of the Human Cell Atlas General Meeting 2023 to Wikidata, hopefully bringing insights on the event and the HCA community.