## Mapping the metascience on bioinformatics onto Wikidata {.page_break_before}

### Introduction

Throughout the development of this project, we recognized Wikidata's potential to represent not only scientific concepts but also scholarly metadata. 
Wikidata serves as more than a platform for capturing information about the world; it also stores structured data on the sources for encoded statements, making it a citation repository containing structured data on tens of millions of scientific articles.
This initiative was led by WikiCite, a project dedicated to improving citations in the context of Wikimedia platforms. 
The citation metadata on Wikidata fuels the platform's provenance system, enabling queries like listing the articles used as sources for cell types in the platform (Figure @fig:cell_type_sources).

![Articles and databases most frequently used for curating cell types on Wikidata](https://raw.githubusercontent.com/lubianat/phd_figures/master/wikidata_cell_curation/sources_for_cell_classes_on_wikidata.png){#fig:cell_type_sources width="85%"}

Wikidata's metascientific scope has progressively broadened, developing an infrastructure to represent other aspects of the publication landscape like theses, preprints, authors, institutions, research topics, academic software, conferences, and more. 
One project, Scholia, emerged as a platform showcasing Wikidata, creating SPARQL-based automatic profiles for different entities in academia. [@wikidata:Q41799194]
Scholia's insights, like co-author graphs and topic scores for institutions, prompted us to think more broadly about the biological knowledge graph in Wikidata, including not only scientific concepts but also the players in the bioknowledge ecosystem.    

Recognizing the platform's potential, we dedicated time to enhance Wikidata's tooling and coverage for bioinformatics information, focusing on the Brazilian bioinformatics community and the Human Cell Atlas context. 
In this chapter, we outline the PyOrcidator package, a tool to integrate information from the ORCID platform to Wikidata on demand, a project to map the Brazilian bioinformatics landscape to Wikidata, a semantic catalog of USP Bioinformatics thesis, and an example of how to leverage Wikidata's infrastructure to gain insights on specific academic events.


### Methods

Our process of building a metascientific knowledge graph and then displaying it to the community consisted of several steps, from data modelling to visualization. 
After identifying an analytical scope, we manually made the core Wikidata entries for entities, adding aspects such as the location and parent institutions of programs, and start and end dates for academic events.  
Then, we used web scraping in Python to recover lists of related people, and manually reconciled the names to Wikidata, using both WDCuration and a set of other tools, such as the Wikipedia and Wikidata add-on for Google Sheets (<https://workspace.google.com/marketplace/app/wikipedia_and_wikidata_tools/595109124715>)and the Open Refine data curation platform (<https://openrefine.org/>)

For each identified person, we pulled the ORCID identifier and run the PyOrcidator tool (described below) to update the information on Wikidata, as well as other tools such as AuthorDisambiguator to disambiguate articles to their authors. 
Finally, to visualize the information, we wrote a series of SPARQL queries and/or used Scholia's standard SPARQL queries, to generate insights and visualizations on the graph.
Due to the ever-updating nature of the curation effort, we made sure that all subprojects had online, live dashboards either independently on GitHub Pages or on the Scholia platform.

#### PyOrcidator

The biomedical research landscape is composed by the interactions and products of several individual researchers.
By having accurate metadata on people, we empower our community to better tailor its projects to the society.
For example, actionable metadata on affiliations, gender and (if public sources are available) ethnicity can help Equity, Diversity and Inclusion (EDI) efforts. 
In the context of the Human Cell Atlas, it can help us identify biases and gaps in publications and events, and thus tweak the actions around the community to help the project to reach its goals. 
It can also help academic departments and programs to know more about its academic output, and use this data for improvement.

By maintaining accurate metadata on individuals, we can better adapt our community projects to society's needs.
Actionable metadata, such as affiliations, gender, and (when publicly available) ethnicity, can significantly aid Equity, Diversity, and Inclusion (EDI) efforts. 
Within the context of the Human Cell Atlas, such data can help identify biases and gaps in publications and events, thereby guiding community actions to assist the project in achieving its goals. 
Moreover, it can enable academic departments and programs to gain insights into their academic output, using this data for improvement.

Currently, ORCID serves as the primary international standardized provider of academic curricula for individuals. [@wikidata:Q30512726] Despite _Currículo Lattes_ being the standard for academic curricula in Brazil, our focus on ORCID is due to its broader international coverage, open Application Program Interface (API), and clear CC0 (public domain dedication) license, which permits the reuse of metadata.
ORCID is also extensively mapped on Wikidata, simplifying the retrieval of ORCID identifiers once the Wikidata ids are known. 
The ORCID API provides semi-structured information for every researcher, including DOIs for listed publications. 
However, much of the information provided is in natural language, where users enter their affiliations and research topics in free-format fields. 
This practice complicates, for instance, querying ORCID for all members of a specific research institution, as names may be represented in various ways. 

PyOrcidator is a tool that merges the parsing of the ORCID API with dictionary handling using WDCuration, allowing curators to map, on demand, the free labels of ORCID to the numeric identifiers of Wikidata.
It maintains three dictionaries: one for roles (e.g., "PhD Student", "Lead developer"), one for institutions (e.g., "University of São Paulo", "FioCruz"), and one for topics (e.g., "Cell biology", "bioinformatics").
Package users can clone the GitHub repository containing these dictionaries, perform curation on demand, and push the changes back to GitHub.
This decentralized curation facilitates the gradual mapping of institution labels to Wikidata, effectively enabling the reuse of curation efforts.

For each ORCID, PyORCIDator generates a file in the Quickstatements format that contains the triples to be updated in the knowledge graph. This includes cross-identifiers present in ORCID (e.g., to Twitter and ResearchGate), affiliation and education information, studied topics, and the author's publications.
The PyORCIDator package is crucial for mapping academic programs and conferences to Wikidata, enabling targeted curation of the metadata for relevant researchers.

### The Landscape of Brazilian Bioinformatics

To exercise Wikidata as a framework for cataloging scientific metadata, we set ourselves to the task of designing a map of the stakeholders in Brazilian bioinformatics. 

We created Wikidata entries for several bioinformatics programs in Brazil, and mapped the researchers affiliated to each of them on Wikidata. 
We also assumed that any researcher affiliated to a core bioinformatics program also had bioinformatics as field of work.
For determining the people in scope of the project, we decided to filter for the following constraints: 

 1. People that have the triple "field of work (P101) --> bioinformatics (Q128570)"
 2. People that have some relation - "employed by (P108)", "affiliated to (P1416)",  "member of (P463)" or "educated at (P63)" - to an institution that has the triple "country (P17) --> Brazil (Q155)". 

 We decided on that loose mapping to reach bioinformaticians in Brazil widely, even those that are not affiliated to a bioinformatics-only academic program. 
 Besides our individual curation of the academic programs, we organized a 2-day virtual event in November 2023 that gathered people from across Brazil to share their perspectives and aid in the curation effort. 

The final product of the mapping is a live dashboard on the landscape of Brazilian bioinformatics that can be accessed at
https://lubianat.github.io/bioinfo_brasil/dashboard, highlighted in Figure @fig:landscape_of_bioinformatics. 
The dashboard contains a series of SPARQL query results showcasing different facets of the Brazilian bioinformatics, such as the giant co-authorship component (Figure @fig:landscape_of_bioinformatics A-B) with few separate communities, and the bias towards institutions in the United States and Europe for study and work. 


![Selected visualizations from the dashboard of the landscape of Brazilian bioinformatics. A) The co-authorship network of all Brazilian bioinformaticians (according to the previously outlined criteria). B) An inset of the graph, showing the high connectivity of the University of São Paulo community. Images are displayed whenever available under an open license in the Wikimedia Commons Database. For researchers without images, women are shown in green and man are shown in orange. Other genders or missing information would appear in white. C) The map of past and present affiliations of Brazilian bioinformaticians. All images displayed are screenshots of SPARQL query results available with live-updated data at <https://lubianat.github.io/bioinfo_brasil/dashboard>.](https://raw.githubusercontent.com/lubianat/phd_figures/master/metascience_on_wikidata/landscape_of_brazilian_bioinformatics.png){#fig:landscape_of_bioinformatics width="70%"} 

#### USP Bioinformatics Thesis

Besides focusing on the bioinformatics researcher, we allocated time to model and gather insights on the theses and dissertations produced by students at the University of São Paulo Graduate Program in Bioinformatics (also available at <https://lubianat.github.io/bioinfo_brasil/dashboard>).
We parsed the metadata for each publication from the "Teses USP" repository into a curation framework, where committee members and topics were transformed from free text into Wikidata identifiers. 
We subsequently created each publication as a new item on Wikidata, incorporating the reconciled metadata, thus making them queryable.

In Figure {@fig:bioinformatics_theses}A, "bioinformatics" emerges as the dominant main topic. However, we also see concepts such as "transcriptome", "metagenomics", and "single-nucleotide polymorphisms", indicating the range of subjects within the program's scope. In @fig:bioinformatics_theses B, we observe that Prof. Helder Nakaya, the supervisor of this project, has attended the most defenses in the program. This data does not distinguish between own students and invitations, hence this potential bias needs consideration.

Figure {@fig:bioinformatics_theses}C showcases a co-participation network in bioinformatics defense committees. 
We hypothesized that, akin to co-authorship graphs, a committee co-participation network could yield insights into the less visible structures of science. 
In this network, smaller, disconnected subcomponents, such as the one highlighted in Figure {@fig:bioinformatics_theses}D, underscore different scientific communities.
In this case, the three professors from the program in the subgraph, Prof. Júlia Pavan, Prof. Anatoli Iambartsev, and Prof. Antonio Galves, all hail from the Institute of Mathematics and Statistics, potentially indicating a difference in the nature of research conducted and evaluated.

Beyond providing factual evidence, the open metascientific maps on Wikidata offer valuable insights into the hidden structures of science.
These maps present a democratic approach to navigating academic affinities, unveiling patterns in co-authorship and committee thesis participation. 
Aside from their apparent applications in enhancing academic structures and detecting fraudulent schemes, these maps also carry significant non-obvious implications. 
By openly encoding the metascientific structure, newcomers and young scientists gain a clearer understanding of the inner dynamics of an academic career, thereby increasing their odds of success.

![Topics and committee members of USP Bionformatics Theses and dissertations. A) The top 10 topics via Teses USP of the thesis in the program. B) The 5 people that most participated in defenses of the program C) A co-participation in committee graph for the USP Bioinformatics program. Researchers affiliated to the program are shown in blue. D) An inset of the graph, showing a community disconnected from the giant component. Graphs are not directional, arrows are an artifact of the graph visualization in the Wikidata Query Service.](https://raw.githubusercontent.com/lubianat/phd_figures/master/metascience_on_wikidata/bioinformatics_theses.png){#fig:bioinformatics_theses width="70%"}


### Event-based curation 

#### Natal Bioinformatics Forum 2023

With the conclusion of the COVID-19 pandemic health emergency, academic conferences have resumed. 
These events form the bedrock of contemporary scientific practice, enabling interaction among individuals from diverse backgrounds and sparking the initiation of new projects. 
Leveraging state-of-the-art Linked Open Data standards to structure event-level information can provide significant benefits to participants, organizers, and funders. For example, co-author graphs of attendees and speakers can identify potential collaborations. Similarly, metadata related to location and gender can help organizers enhance event diversity.
In this report, we demonstrate the application of a knowledge graph using the speakers' network at the Natal Bioinformatics Forum, an academic conference held in Natal, Rio Grande do Norte, in March 2023.

We scraped the speakers' information, reconciled it with Wikidata, curated it on the platform, and made it accessible through a dashboard at <https://lubianat.github.io/natal_bioinformatics2wikidata/>. 
In Figure {@fig:natal_bioinformatics}A, we can view the co-authorship relations among the event's speakers. 
Prof. Ana Tereza Vasconcelos appears as a central figure in the graph, serving as a connecting hub across various subcomponents. The importance of these relationships was highlighted during her presentation at the event. Prof. Ana Tereza is acknowledged as one of the pioneers in the field of bioinformatics in Brazil and has been actively publishing in this field for several decades, forming extensive collaborations throughout her career.

The reconlitiation the event to Wikidata also enabled a quick overview of the types of software used by the speakers in their publications as one can see in Figure {@fig:natal_bioinformatics}B.
Apart from the imaging software ImageJ, we noticed a preference for R packages dealing with transcriptomics, such as "edgeR", "DESeq2", "limma", and "affy". This also sheds light on the community's inner practices. 

Lastly, the map of the speakers' affiliations in Figure {@fig:natal_bioinformatics}C reveals once more a clear bias towards the United States and Europe. 
Remarkably, Latin American institutions were absent from the speakers' academic history. Even within Brazil, we observe two localized clusters: one in the south-southwest and the other in a small part of the northwest.
These affiliation maps are potent tools for identifying geographic biases and, hopefully, for addressing them in future events.


![An overview of the speakers at the Natal Bioinformatics Forum 2023. A) The co-authorship network of the speakers at the event, excluding speakers with no connections. B) A bubble plot showing the pieces of software most used by the spearkers in publications, according to Wikidata. C) The map of past and present affiliations of the speakers at the event.](https://raw.githubusercontent.com/lubianat/phd_figures/master/metascience_on_wikidata/natal_bioinformatics.png){#fig:natal_bioinformatics width="70%"} 


#### Contextualization with the Human Cell Atlas

The metascientific analysis outlined here is not only relevant to Brazilian bioinformatics, but also to understanding and addressing gaps within the Human Cell Atlas. 
We have begun curating the metadata of Human Cell Atlas Publications (<https://www.humancellatlas.org/publications/#>, <https://w.wiki/6hzz>) on Wikidata, offering a novel approach to scrutinize biases (geographic and others) within the project. 
This is particularly significant in the context of the HCA, as one of its core objectives is to construct an atlas that accurately represents human diversity. 
Furthermore, we are in the process of mapping the speakers and poster presenters of the Human Cell Atlas General Meeting 2023 on Wikidata, with the goal of gaining insights into the event and the HCA community.
